static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -112,7 +113,9 @@ function DomMutations({replay}: Props) {
               </UnstyledButton>
             </MutationDetailsContainer>
             <CodeContainer>
-              <HTMLCode code={html} />
+              <CodeSnippet language="html">
+                {beautify.html(html, {indent_size: 2})}
+              </CodeSnippet>
```

Comment:
@Jesse-Box 
By default `CodeSnippet` displays a copy button on hover:
<img width="1133" alt="Screen Shot 2022-09-30 at 3 14 13 PM" src="https://user-images.githubusercontent.com/44172267/193363580-9ce457df-c892-40ee-a217-602639b8db02.png">

Is a copy button useful here? If not I can remove it (using the `hideCopyButton` prop)